components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/analytics/SynapseAnalyticsDataProvider.java

```diff
@@ -163,6 +168,49 @@ public API getApi() throws DataNotFoundException {
             api.setApiVersion(apiObj.getApiVersion());
             api.setApiCreator(apiObj.getApiProvider());
             api.setApiCreatorTenantDomain(MultitenantUtils.getTenantDomain(api.getApiCreator()));
+            List<URITemplate> uriTemplates = new ArrayList<>();
+
+            for (URLMapping uriTemplate : apiObj.getUrlMappings()) {
+                org.wso2.carbon.apimgt.common.analytics.publishers.dto.URITemplate uriTemplateObj
+                        = new org.wso2.carbon.apimgt.common.analytics.publishers.dto.URITemplate();
+                if (uriTemplate.getHttpMethod() != null && uriTemplate.getHttpMethod()
+                        .equals(messageContext.getProperty("api.ut.HTTP_METHOD")) && uriTemplate.getUrlPattern() != null
```

Comment:
You can use the constant[1] here.

[1] https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/APIMgtGatewayConstants.java#L33